### DISCUSSION TITLE
Domain certificates not working: ERR_CERT_AUTHORITY_INVALID

### DISCUSSION THREAD

**Discussion by sudhanshug16** [Dokploy/dokploy#473] [General]
Hey, we had dokploy setup on our aws instance and it was working fine. 
But it stopped working when we created a new instance in Hetzner yesterday. We assigned the domain "preview.noscrubs.dev" to the server.

Port 80 and 443 are open. We have an A record which resolves to 37.27.199.114 (the static ip of the server). Here is a screenshot showing the setup:
<img width="743" alt="image" src="https://github.com/user-attachments/assets/01381ea2-4d86-47a6-b61f-44b3676f3716">

We are getting the same error if we assign more domains to services. We have tried uninstalling and reinstalling dokploy and the same still happens.

**Comment by sudhanshug16** [comment]
`/etc/dokploy/traefik/dynamic/acme.json` is empty

**Reply by Siumauricio** [reply]
@sudhanshug16 Can you try to restart traefik?

**Comment by sudhanshug16** [comment]
I deleted the VPS and created a new one - that worked. But restarting traefik did not work.
